pkg/daemon/ceph/config/config.go

```diff
@@ -27,17 +27,31 @@ import (
 	"github.com/go-ini/ini"
 	"github.com/rook/rook/pkg/clusterd"
 	"github.com/rook/rook/pkg/daemon/ceph/client"
+	"github.com/rook/rook/pkg/util"
 )
 
 var logger = capnslog.NewPackageLogger("github.com/rook/rook", "cephconfig")
 
+// ConfigDir should return the Ceph configuration directory. This should be left as-is for Rook
+// during runtime, but for unit tests, this can be overridden to allow tests to read/write to a
+// temporary testing dir safely.
+var ConfigDir func() string
+
+// RunDir should return the run directory for a particular Ceph daemon. Like the conf dir
+// builder, this can be overridden to allow tests to read/write to a temporary testing dir safely.
+var RunDir func() string
```

Comment:
Oh. Do you mean that it could be like below? I suppose this would make more sense. I was probably overthinking this.
```go
var RunDir string

func init() {
    RunDir = "/var/lib/ceph"
}
```

Reply:
Yes, that would effectively be the same.